modules/mobile/docs/openapi.yaml

```diff
@@ -1477,6 +1477,45 @@ paths:
       security:
         - Bearer: []
       summary: /v0/health/immunizations
+  /v0/health/allergy-intolerances:
+    get:
+      description: >
+        Retrieves a list of the user's known allergies related to medication, food, or other substances
+      operationId: list_allergy_intolerances
+      parameters:
+        - description: The number of resources that should be returned in a single page. The maximum count size is 100. Defaults to 30.
```

Comment:
we now have a standard way of doing pagination for our endpoints with `pagination_helper`, page[number] and page[size]. May want to use that in our controller and then convert it to this single count parameter for the client.

Reply:
I should probably remove the count because the client isn't actually using the count for anything

Reply:
Updated docs to show no parameters are actually used

Reply:
oh yeah, may want to hardcode that value to 100 to minimize the number of calls the client has to make. I can't imagine that people have more than 100 allergies but maybe it's a thing? probably not worth implementing pagination since the number probably doesn't get bigger than 100

Reply:
It is hardcoded in the client